### ISSUE TITLE
re-enable logging

### ISSUE THREAD

**Issue by mmmaaatttttt** [rithmschool/numbers_api#10]
there's a function that used to be part of the project which would log every request (https://github.com/rithmschool/numbers_api/blob/5f03fff7e23a08ee578ab38db3d77f6a0e974b6e/routes/api.js#L24) and occasionally write these logs to a file. There were also two endpoints that would reveal summary statistics based on these log files: https://github.com/rithmschool/numbers_api/blob/master/app.js#L143-L149

However, the logging was consuming >100% CPU, mostly likely because the files were getting quite large and so reading / writing was expensive. Would be nice to bring this feature back, but we should think about how to better support it at current scale. (Redis? DB? something else?)

**Comment by maxnawa31** [comment]
would it be possible to write logs to a db, and have some sort of cron job that deletes from the table at some time period? This is under the assumption that we don't need to be saving every single log

**Comment by mmmaaatttttt** [comment]
@maxnawa31 yup for sure, originally it would write to a file and then occasionally just make a new file, but the file i/o operations were taking up a ton of cpu. i'm fine with adding a persistence layer, i just don't want to spend a bunch of money on it.

**Comment by maxnawa31** [comment]
could we maybe remove the file altogether, and instead write to an in-memory store (i.e. Redis) and then at some point write to the database? I suppose this would follow a 'Write-Back' protocol

**Comment by mmmaaatttttt** [comment]
yeah i'd be super down to explore this option @maxnawa31

**Comment by mmmaaatttttt** [comment]
@Nalipp I think this is a promising direction:

https://www.npmjs.com/package/winston
https://www.digitalocean.com/community/tutorials/how-to-use-winston-to-log-node-js-applications
